### DISCUSSION TITLE
CPM and PackageDownload and NU1505

### DISCUSSION THREAD

**Discussion by lonix1** [NuGet/Home#12036] [Q&A]
(From my original comment in [this](https://github.com/dotnet/sdk/issues/24747#issuecomment-1218051535) thread.)

I'm using `6.0.400` (i.e. latest). I'm using "Central Package Management".

Repro:

`MyProject/Directory.Packages.props`:
````xml
<ItemGroup>
  <PackageDownload Include="minver-cli" Version="[4.1.0]" />
</ItemGroup>
````

`MyProject/MyProject.csproj`:
````xml
<ItemGroup>
  <PackageDownload Include="minver-cli" />
</ItemGroup>
````

`dotnet build`:
> /home/username/code/MyProject/MyProject.csproj : warning NU1505: Duplicate 'PackageDownload' items found. Remove the duplicate items or use the Update functionality to ensure a consistent restore behavior. The duplicate 'PackageDownload' items are: minver-cli [4.1.0], minver-cli .

Someone commented that this is not how CPM is supposed to work. There's no much info on CPM as it's still new. It works for me for the normal case with `PackageReference`.

Does CPM work with `PackageDownload` as well? If yes, how do I fix my syntax? Thanks.

**Comment by erdembayar** [comment]
Fyi @jeffkl

**Comment by jeffkl** [comment]
Central Package Mangement does not currently work with `PackageDownload`.  You can get the similar behavior of `PackageDownload` with the following:

**MyProject.csproj**
```xml
<ItemGroup>
  <PackageReference Include="minver-cli" IncludeAssets="None" />
</ItemGroup>
```

**Directory.Packages.props:**
```xml
<ItemGroup>
  <PackageVersion Include="minver-cli" Version="4.1.0" />
</ItemGroup>
```
